Administrator Guide/arbiter-volumes-and-quorum.md

```diff
@@ -0,0 +1,219 @@
+describes how to resolve files that are in split-brain using gluster cli or the
+mount point. Almost always, split-brains occur due to network disconnects (where
+a client temporarily loses connection to the bricks) and very rarely due to
+the gluster brick processes going down or returning an error.
+
+Server-quorum and some pitfalls
+================================
+http://www.gluster.org/community/documentation/index.php/Features/Server-quorum
+provides a detailed description of this feature.
+The volume options for server-quorum are:
+
+> Option:cluster.server-quorum-ratio                                              
+  Value Description: 0 to 100           
+ 
+> Option:cluster.server-quorum-type                                                  
+  Value Description:  none | server                                                  
+ If set to server, this  option enables the specified volume to participate in the server-side quorum.
+ If set to none, that volume alone is not considered for volume checks.
+
+The cluster.server-quorum-ratio is a percentage figure and is cluster wide- i.e.
+you cannot have a different ratio for different volumes in the same trusted pool.
+
+For a two-node trusted storage pool it is important to set this value to be
+greater than 50% so that two nodes separated from each other do not both believe
+they have quorum simultaneously. For a 2 node plain replica volume, this would
+mean both nodes need to be up and running. So there is no notion of HA/failover.
+
+There are users that create a replica 2 volume from 2 nodes and peer-probe
+a 'dummy' node without bricks and enabling server quorum with a ratio of 51%.
+This does not prevent files from getting into split-brain. For example, if B1
+and B2 are the bricks/nodes of the replica and B3 is the dummy node, we can
+still end up in split-brain like so:
+
+1. B1 goes down, B2 and B3 are up. server-quorum is still. File is modified
+by the client.
+2. B2 goes down, B1 comes back up. Server-quorum is met. Same file is modified
+by the client.
+3. We now have different contents for the file in B1 and B2 ==>split-brain.
+
+In the author’s opinion, server quorum is useful if you want to avoid split-brains 
+to the volume(s) configuration across the nodes and and not in the I/O path.
+Unlike in client-quorum where the volume becomes read-only when quorum is lost, loss of
+server-quorum in a particular node makes glusterd kill the brick processes on that
+node (for the participating volumes) making even reads impossible.
+
+Client-quorum
+==============
+Client-quorum is a feature implemented in AFR to prevent split-brains in the I/O
+path for replicate/distributed-replicate volumes. By default, if the client-quorum
+is not met for a particular replica subvol, it becomes read-only. The other subvols
+(in a dist-rep volume) will still have R/W access.
+
+The following volume set options are used to configure it:
+>Option: cluster.quorum-type  
+  Default Value: none
```

Comment:
This is rendered in a single line. You may want to add two spaces at the end of the line (equivalent of /n in markdown)